<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QRadioButton" name="rtMode">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>130</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gisha&quot;;
color: rgb(130, 255, 108) ; </string>
   </property>
   <property name="text">
    <string>Real Time Mode</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="testMode">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Gisha&quot;;
color: rgb(130, 255, 108) ; </string>
   </property>
   <property name="text">
    <string>Test Mode</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadECG">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;load&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);
background-color: rgb(248, 248, 248);
background-color: rgb(91, 91, 91);

border-color: rgb(255, 255, 255);
color: rgb(130, 255, 108) ; 
font: 14pt &quot;Gisha&quot;;</string>
   </property>
   <property name="text">
    <string>Load ECG</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="rawDataView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>971</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(26, 255, 23);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>430</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber{
    color: rgb(130, 255, 108) ;  
    
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>1</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="PlotWidget" name="modDataView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>971</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>proccesed data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>raw data</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="start">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>336</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
font: 14pt &quot;Gisha&quot;;
background-color: rgb(255, 255, 255);
color: rgb(130, 255, 108) ; 
selection-background-color: rgb(93, 93, 93);</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadGSR">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;load&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);
background-color: rgb(248, 248, 248);
background-color: rgb(91, 91, 91);
border-color: rgb(255, 255, 255);
font: 14pt &quot;Gisha&quot;;
color: rgb(130, 255, 108) ;  
</string>
   </property>
   <property name="text">
    <string>Load GSR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>310</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(91, 91, 91);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>310</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;load&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);
background-color: rgb(248, 248, 248);
background-color: rgb(91, 91, 91);
border-color: rgb(255, 255, 255);
font: 14pt &quot;Gisha&quot;;
color: rgb(130, 255, 108) ;  
</string>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>410</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>real-time workeload</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>1122</x>
     <y>490</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="CL">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>-10</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>91</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;AR ESSENCE&quot;;
color: rgb(179, 179, 179);
color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;COGNITIVE WORKELOAD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="predictor">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>50</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 42pt &quot;AR ESSENCE&quot;;
color: rgb(236, 236, 236);

</string>
   </property>
   <property name="text">
    <string>PREDICTOR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RTinterface">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;load&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);
background-color: rgb(248, 248, 248);
background-color: rgb(91, 91, 91);
border-color: rgb(255, 255, 255);
font: 14pt &quot;Gisha&quot;;
color: rgb(130, 255, 108) ;  
</string>
   </property>
   <property name="text">
    <string>Load Real Time Stream</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadEvents">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>200</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;load&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);
background-color: rgb(248, 248, 248);
background-color: rgb(91, 91, 91);
border-color: rgb(255, 255, 255);
font: 14pt &quot;Gisha&quot;;
color: rgb(130, 255, 108) ;  
</string>
   </property>
   <property name="text">
    <string>Load Events Log</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>971</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="stress_level">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>530</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1016</x>
     <y>495</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>Stress Level</string>
   </property>
  </widget>
  <zorder>CL</zorder>
  <zorder>pushButton</zorder>
  <zorder>rtMode</zorder>
  <zorder>testMode</zorder>
  <zorder>loadECG</zorder>
  <zorder>modDataView</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>start</zorder>
  <zorder>loadGSR</zorder>
  <zorder>rawDataView</zorder>
  <zorder>reset</zorder>
  <zorder>label_3</zorder>
  <zorder>progressBar</zorder>
  <zorder>predictor</zorder>
  <zorder>RTinterface</zorder>
  <zorder>loadEvents</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>stress_level</zorder>
  <zorder>label_4</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
